test/common_utils.py

```diff
@@ -23,6 +23,7 @@
 IS_PY39 = sys.version_info.major == 3 and sys.version_info.minor == 9
 PY39_SEGFAULT_SKIP_MSG = "Segmentation fault with Python 3.9, see https://github.com/pytorch/vision/issues/3367"
 PY39_SKIP = unittest.skipIf(IS_PY39, PY39_SEGFAULT_SKIP_MSG)
+IN_CIRCLE_CI = os.getenv("CIRCLECI", False) == 'true'
```

Comment:
nit: doing the following will make it consistent with other env vars
```suggestion
IN_CIRCLE_CI = os.getenv("CIRCLECI", '0') == '1'
```

Reply:
CircleCI sets this variable to 'true' according to their docs, it's not something we can change I think: https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables


Also, setting the default to '0' would still evaluate the variable to True even when it's not set: 

```py
>>> bool('0')
True
```

Which makes me wonder whether the other global variables
```py
ACCEPT = os.getenv('EXPECTTEST_ACCEPT', '0') == '1'
TEST_WITH_SLOW = os.getenv('PYTORCH_TEST_WITH_SLOW', '0') == '1'
```

are properly defined?

Reply:
I see, if you don't have access to change them that's fine.  

The `PYTORCH_TEST_WITH_SLOW` is set correctly in the bash scripts. See:

https://github.com/pytorch/vision/blob/19ad0bbc5e26504a501b9be3f0345381d6ba1efc/.circleci/unittest/windows/scripts/run_test.sh#L8

https://github.com/pytorch/vision/blob/19ad0bbc5e26504a501b9be3f0345381d6ba1efc/.circleci/unittest/linux/scripts/run_test.sh#L8

The `EXPECTTEST_ACCEPT` is manually set by the developer when they add a new expect file, so you won't find it in the repo. 

Another option is to change the other env vars to be consistent then. I'm fine leaving them as is or changing them.

Reply:
My comment about `PYTORCH_TEST_WITH_SLOW` and `ACCEPT` was about the default value passed to `getenv`, that is `'0'`.

But I realize now that those variable will still be equal to `False` when they're not set, not to `'0'`, since the comparison will be `'0' == '1'`.

I guess we could leave the default value to its default (None), and just do

```py
ACCEPT = os.getenv('EXPECTTEST_ACCEPT') == '1'
etc.
```

 but it doesn't really matter.